src/js/components/Settings/SettingsDomain.jsx

```diff
@@ -175,10 +175,10 @@ class SettingsDomain extends Component {
     const { classes } = this.props;
 
     return (
-      <Wrapper>
+      <div>
```

Comment:
Hi @jarodpeachey, in general, I'm moving towards giving divs names (even if there aren't style associated with them yet) because I think it make the code easier to maintain, and then also easier to add styles if we need to make changes directly within the component in the future.